### PR TITLE
[database]: Remove hard/soft limits for client-output-buffer in redis

### PR DESCRIPTION
**Pull request by pavel-shirshov**
Temporary/fast fix for https://github.com/Azure/sonic-sairedis/issues/176

### CODE REVIEW

**Comment by stcheng** [comment]
This is to solve https://github.com/Azure/sonic-sairedis/issues/176.

**Comment by pavel-shirshov** [comment]
Please don't merge it until it's done testing

**Comment by pavel-shirshov** [comment]
It's ready to be merged

File: dockers/docker-database/Dockerfile.j2 [hunk 1]
```diff
@@ -28,7 +28,8 @@ RUN sed -ri 's/^(save .*$)/# \1/g;
              s/^daemonize yes$/daemonize no/;                                           \
              s/^logfile .*$/logfile ""/;                                                \
              s/^# syslog-enabled no$/syslog-enabled no/;                                \
-             s/^# unixsocket/unixsocket/                                                \
+             s/^# unixsocket/unixsocket/;                                               \
+             s/^client-output-buffer-limit pubsub 32mb 8mb 60/client-output-buffer-limit pubsub 0 0 0/ \
```
Comment (lguohan) [inline dockers/docker-database/Dockerfile.j2]: use regex \d+mb \d+mb \d+?

Comment (pavel-shirshov) [inline dockers/docker-database/Dockerfile.j2]: ok

Comment (pavel-shirshov) [inline dockers/docker-database/Dockerfile.j2]: Done. \d didn't work for sed -ir

**Pull request merged**